commit 05ec362b6a5984c8664818c9d6865e6a811f9973
Author: Friedrich W. H. Kossebau <kossebau@kde.org>
Date:   Tue Feb 4 20:54:38 2014 +0100

    Fix wrong include of addressrangelist.h (not part of installed headers)

diff --git a/gui/bytearraytableranges.h b/gui/bytearraytableranges.h
index 9f508a6..4d0f148 100644
--- a/gui/bytearraytableranges.h
+++ b/gui/bytearraytableranges.h
@@ -28,7 +28,7 @@
 #include "coordrangelist.h"
 #include "oktetagui_export.h"
 // Okteta core
-#include "addressrangelist.h"
+#include "addressrange.h"
 
 
 namespace Okteta
